Misc/NEWS.d/next/IDLE/2019-05-05-16-27-53.bpo-13102.AGNWYJ.rst

```diff
@@ -0,0 +1 @@
+When saving a file, call os.flush() so bits are flushed to e.g. USB drive.
```

Comment:
This is a typo, I think, should say call `os.fsync()`.

I forgot to mention this when I reported